moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js

```diff
@@ -100,27 +127,35 @@ const ProjectSummaryKnackDataTrackerSync = ({
         .then(result => {
           if (result.errors) {
             // Successful HTTP request, but knack indicates an error with the query, such as non-existent ID.
-            // Reject the promise to fall through to the .catch() method
-            return Promise.reject(result);
+            // Throw error & cause the promise to fall through to the .catch() method
+            throw result;
           } else {
             // Successful HTTP request with meaningful results from Knack
             project.currentKnackState = result; // this assignment operates on `project` which is defined in broader scope than this function
-            return fetch(knackEndpointUrl, {
-              // fetch returns a Promise for the next step
-              method: knackHttpMethod,
-              headers: buildHeaders,
-              body: buildBody(),
-            });
+
+            Promise.resolve(numberOfSignalsInProject)
```

Comment:
I was fussing around with wrapping this up in some sort of function to avoid the explicit Promise.resolve() call, but on consideration, the exact thing I needed was a resolved promise, so I put this in here. 

Open to any suggestions if this not the best route.